protected/controllers/BaseAdminController.php

```diff
@@ -225,7 +225,7 @@ protected function genericAdmin($title, $model, array $options = array(), $key =
                                         $message = "Deleting Error: disorder group '".$item->name."' is in use";
                                         break;
                                     default:
-                                        $message = "Operation failed due to error ".$excep->getCode();
+                                        $message = "Operation failed, please contact support team for help!";
```

Comment:
Not sure if the exclamation mark is necessary here, but ultimately it comes down to user experience.